aws-event-sources-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-event-sources-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: aws-event-sources-operator
+  template:
+    metadata:
+      labels:
+        name: aws-event-sources-operator
+    spec:
+      serviceAccountName: aws-event-sources-operator
+      containers:
+        - name: aws-event-sources-operator
+          image: gcr.io/triggermesh/aws-event-sources-operator:v0.0.1
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "aws-event-sources-operator"
```

Comment:
No quotes

Reply:
This is a generated line. I will leave it as is